src/browser_action/metric.js

```diff
@@ -68,13 +68,18 @@ export class Metric {
   }
 
   toLocaleFixed({value, unit, precision}) {
-    return value.toLocaleString(undefined, {
-      style: unit && 'unit',
-      unit,
-      unitDisplay: 'short',
-      minimumFractionDigits: precision ?? this.digitsOfPrecision,
-      maximumFractionDigits: precision ?? this.digitsOfPrecision
-    });
+    // Ideally we'd use toLocaleString but that has performance implications
+    // (see https://github.com/GoogleChrome/web-vitals-extension/issues/107)
+    let formattedUnit = '';
+    switch (unit) {
+      case 'second':
+        formattedUnit = ' s';
+        break;
+      case 'millisecond':
+        formattedUnit = ' ms';
+        break;
+    }
+    return value.toFixed(precision || this.digitsOfPrecision) + formattedUnit;
```

Comment:
should this be
```suggestion
    return value.toFixed(precision ?? this.digitsOfPrecision) + formattedUnit;
```
for when `precision` is `0`?

Reply:
Oh great spot. Needed for INP and FID.